tempesta_fw/http_msg.c

```diff
@@ -79,7 +140,6 @@ __http_msg_hdr_val(TfwStr *hdr, unsigned id, TfwStr *val, bool client)
 		[TFW_HTTP_HDR_X_FORWARDED_FOR] = SLEN("X-Forwarded-For:"),
 		[TFW_HTTP_HDR_KEEP_ALIVE] = SLEN("Keep-Alive:"),
 		[TFW_HTTP_HDR_TRANSFER_ENCODING] = SLEN("Transfer-Encoding:"),
-		[TFW_HTTP_HDR_USER_AGENT] = SLEN("User-Agent:"),
```

Comment:
It looks confusing to init the same array member twice, since
https://github.com/tempesta-tech/tempesta/blob/81b45c2eb6b7682d934ea821c41dad6b2d24387c/tempesta_fw/http.h#L186

The `User-agent` case is checked in a few lines below:
https://github.com/tempesta-tech/tempesta/blob/81b45c2eb6b7682d934ea821c41dad6b2d24387c/tempesta_fw/http_msg.c#L100